src/commands/config/handler.js

```diff
@@ -1,14 +1,25 @@
+const { logger } = require('./../../logger');
+
 const help = require('./help');
 const channel = require('./channel');
 const cron = require('./cron');
 const embeds = require('./embeds');
 const interval = require('./interval');
 const state = require('./state');
+const permission = require('./permission');
+const topHandler = require('./../handler');
```

Comment:
not sure why the top handler has to take responsibility for permissions checking.

Reply:
I think this is more convenient to keep command related code on the command handler, and since we have multiple handlers I had to pick the top one to hold this code.